test/run

```diff
@@ -14,6 +14,7 @@ case $TEST_SCENARIO in
     verify|devel|firefox|firefox-devel)
         [ "${TEST_SCENARIO}" = "${TEST_SCENARIO%%devel}" ] || export NODE_ENV=development
         [ "${TEST_SCENARIO}" = "${TEST_SCENARIO##firefox}" ] || export TEST_BROWSER=firefox
+        [ "${TEST_SCENARIO}" = "${TEST_SCENARIO##mobile}" ] || export TEST_MOBILE=""
```

Comment:
This needs to be set to an actual value.

Reply:
Fixed.